conductr/tasks/install-core.yml

```diff
@@ -18,6 +18,14 @@
     - python-dev
     - python-setuptools
 
+- name: Get local (private) ipv4
+  get_url: url=http://169.254.169.254/latest/meta-data/local-ipv4 dest={{ REMOTE_WORK_DIR }}/local_ipv4.txt
```

Comment:
The address is specific to EC2
http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
I expect the whole block changes for non-AWS branches